lib/Component/VCalendar.php

```diff
@@ -37,15 +37,15 @@ class VCalendar extends VObject\Document
      * @var array
```

Comment:
Could we use a typehint like array<class-string> or similar now?

Reply:
Yes, you can use them (as you suggested: `array<string, class-string>`), but they won't be enforced until phpstan level 3. Therefore, I doubt it is useful to add them now.

I have done some preliminary work to increase the phpstan level further, and when doing that we'll soon come across phpstan's demand to specify array's like you suggested.

Reply:
I see. Thx for the clarification